packages/api/server.js

```diff
@@ -33,6 +36,18 @@ global.headCollection = process.env.HEAD_COLLECTION || 'testing-buildsget';
 
 app.use(cors());
 app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true })); // support encoded bodies
+app.use(
+  session({
+    store: new FirestoreStore({
+      dataset: new Firestore(),
+      kind: 'express-sessions'
+    }),
+    secret: 'my-secret',
```

Comment:
Should all sessions have the same secret, which is stored in Secret Manager? Or should I generate a new uuid for each one and store it?

Reply:
We should have one secret that's used to encrypted the session, this will be different than the `github-client-secret`.

[`SESSION_SECRET`](https://www.npmjs.com/package/express-session#secret) is a secret we control, which is used to sign the session delivered to the frontend, such that folks can't easily mess with the data we store in the session.